assets/javascripts/modules/analytics/analyticsEnabled.js

```diff
@@ -0,0 +1,19 @@
+define([
+    'utils/cookie'
+], function (cookie) {
+    'use strict';
+
+    var analyticsEnabled = (
+        window.guardian.analyticsEnabled &&
+        !navigator.doNotTrack &&
+        !cookie.getCookie('ANALYTICS_OFF_KEY')
+    );
+
+    return function(cb){
+        return function() {
+            if (analyticsEnabled) {
```

Comment:
Either way works for me, the only benefit I can see this way is that if we wanted to toggle tracking for events that load after page loaded then we could. I dont think there a strong argument either way.